docs/articles/baseplatform/variant-concept-in-yocto.rst

```diff
@@ -21,7 +21,7 @@ Recommended variant management
 
 The recommended way to handle variants is to keep one directory per variant in
 your layer. The variant directories should go into
-``<meta-yourlayer>/conf/<variant name>``. Each variant should supply a
+``<meta-yourlayer>/conf/variant/<variant name>``. Each variant should supply a
```

Comment:
I'd prefer the shorter path, as it was.. How set are you on this? ;)

Reply:
The shorter path doesn't reflect the structure with what was in the directory before, @tobsan had a good point there.